objects/src/notes/mod.rs

```diff
@@ -249,3 +249,15 @@ impl<'de> serde::Deserialize<'de> for Note {
         Self::read_from_bytes(&bytes).map_err(serde::de::Error::custom)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::Note;
+
+    fn check_if_send<T: Send>() {}
+
+    #[test]
+    fn note_is_send() {
+        check_if_send::<Note>();
+    }
+}
```

Comment:
As long as `ProvenTransaction` contains `Note` as part of its fields we might not need this (ie, this would only be needed for the top-level structs that are passed around tasks and I'm not sure `Note` is one of them but I might  be missing something)

Reply:
removed